test/test_ops.py

```diff
@@ -1333,6 +1333,7 @@ def _tensor_requires_grad(x):
         self.fail(msg)
 
 
+@skipIfSlowGradcheckEnv
 class TestCompositeCompliance(TestCase):
```

Comment:
Why do we run tests other than slow gradcheck tests during slow gradcheck CI?

Reply:
> 1. Why do we run tests other than slow gradcheck tests during slow gradcheck CI?

Yup, completely agree here. Tests that use gradcheck are spread across different test classes, so there's some trickiness here though. As a next step, it might be a good idea to have an allow list of the ones that we know use gradcheck. I originally hesitated because I didn't have an exhaustive list, and I only looked at the test classes that take the longest to run.

2. TestCompositeCompliance is being skipped here because it doesn't use gradcheck at all